src/middlewared/middlewared/migration/0009_system_global_id.py

```diff
@@ -0,0 +1,10 @@
+import uuid
+
+
+async def migrate(middleware):
+    await middleware.call(
```

Comment:
Is this guaranteed this will only ever be called once for the lifetime of the install? Either way, we should probably check if there is a UUID before inserting here always.

Reply:
okay